cmd/scollector/main.go

```diff
@@ -266,6 +266,7 @@ func main() {
 	}
 	collectors.DefaultFreq = time.Second * time.Duration(freq)
 	collect.Freq = time.Second * time.Duration(freq)
+	collect.Tags = opentsdb.TagSet{"os": runtime.GOOS}
```

Comment:
I think that is done by the check function. It is working on devbosun so something is adding it for us.